packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -757,6 +757,8 @@ export async function startBrowser(
       // NOTE: The version of chromedriver or geckodriver in play might also be
       // relevant.
       browserVersion: 'latest',
+      'goog:chromeOptions': { args: ['--start-maximized', ''] },
+      'moz:firefoxOptions': { args: ['--start-maximized', ''] },
```

Comment:
It seems like firefox doesn't implement --start-maximized yet. But I'm keep it here as the ideal for now anyway.